services/afl/patches/afl-cmin.patch

```diff
@@ -0,0 +1,122 @@
+diff --git a/afl-cmin b/afl-cmin
+index a88460a8..b106560f 100755
+--- a/afl-cmin
++++ b/afl-cmin
+@@ -258,7 +258,7 @@ BEGIN {
+   # sanity checks
+   if (!prog_args[0] || !in_dir || !out_dir) usage()
+ 
+-  target_bin = prog_args[0] 
++  target_bin = prog_args[0]
+ 
+   # Do a sanity check to discourage the use of /tmp, since we can't really
+   # handle this safely from an awk script.
+@@ -330,14 +330,18 @@ BEGIN {
+     target_bin = tnew
+   }
+ 
+-  if (0 == system ( "grep -aq AFL_DUMP_MAP_SIZE " target_bin )) {
+-    echo "[!] Trying to obtain the map size of the target ..."
+-    get_map_size = "AFL_DUMP_MAP_SIZE=1 " target_bin
+-    get_map_size | getline mapsize
+-    close(get_map_size)
+-    if (mapsize && mapsize > 65535 && mapsize < 100000000) {
+-      AFL_MAP_SIZE = "AFL_MAP_SIZE="mapsize" "
+-      print "[+] Setting "AFL_MAP_SIZE
++  if (!nyx_mode) {
++    if (!ENVIRON["AFL_MAP_SIZE"] && 0 == system ( "grep -aq AFL_DUMP_MAP_SIZE " target_bin )) {
++      echo "[!] Trying to obtain the map size of the target ..."
++      get_map_size = "AFL_DUMP_MAP_SIZE=1 " target_bin
++      get_map_size | getline mapsize
++      close(get_map_size)
++      if (mapsize && mapsize > 65535 && mapsize < 100000000) {
++        AFL_MAP_SIZE = "AFL_MAP_SIZE="mapsize" "
++        print "[+] Setting "AFL_MAP_SIZE
++      }
++    } else {
++      AFL_MAP_SIZE = "AFL_MAP_SIZE=" ENVIRON["AFL_MAP_SIZE"] " "
+     }
+   }
+ 
+@@ -348,6 +352,8 @@ BEGIN {
+     }
+   }
+ 
++  AFL_PRELOAD = ("AFL_PRELOAD" in ENVIRON) ? ENVIRON["AFL_PRELOAD"]" " " : ""
```

Comment:
you have a stray `"` after `ENVIRON["AFL_PRELOAD"]`

Reply:
Fixed in [c967d03](https://github.com/MozillaSecurity/orion/pull/618/commits/c967d03aea9283e216973fca0481240709ca2141)